include/parser_interface.hpp

```diff
@@ -39,6 +39,21 @@ class ParserInterface
         return types::DbusVariantType();
     }
 
+    /**
+     * @brief API to write keyword's value on hardware.
+     *
```

Comment:
could you add a brief on why this is virtual and what happens

Reply:
done